client/src/Components/common/ProductModal.jsx

```diff
@@ -15,22 +15,20 @@ function ProductModal({ open, onClose, product }) {
 
   const style = {
     position: 'absolute',
-    top: '50%',
-    left: '50%',
-    transform: 'translate(-50%, -50%)',
+    top: '20%',
+    left: '15%',
     width: '70%',
     height: '60%',
```

Comment:
instead using `transform / translate`, consider:
top = (100% - height) / 2
left = (100% - width) / 2

Reply:
I feel like translate affect some behavior of Modal (e.g. click outside the modal does not close the modal)